vmngclient/api/templates/payloads/cisco_vpn_interface_ethernet/feature/cisco_vpn_interface_ethernet.json.j2

```diff
@@ -34,29 +45,18 @@
             "vipValue": 1,
             "vipVariableName": "vpn_if_ipv4_dhcp_distance"
           }
-        }
-      {% endif %} 
-    },
-    "if-name": {
-      "vipObjectType": "object",
-      "vipType": "constant",
-      "vipValue": "{{ interface_name.type.value }}{% if interface_name.number is not none %}{{ interface_name.number }}{% endif %}",
-      "vipVariableName": "vpn_if_name"
-    },
-    "description": {
-      "vipObjectType": "object",
-      "vipType": "ignore",
-      "vipVariableName": "vpn_if_description"
-    },
-    "secondary-address": {
```

Comment:
secondary-address is not in order

Reply:
secondary-address is in ip
![image](https://user-images.githubusercontent.com/73225607/227142251-71f0b21e-2cbd-43ec-b70d-887f8b2e5ee2.png)